spec/rental_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../classes/rental'
+require_relative '../classes/book'
+describe Rental do
+  before :each do
+    @book = Book.new('Common sense approach to programming', 'Micheal Hart')
+
+    @teacher = Teacher.new('math', 19, 'sam')
+    @rental = Rental.new('2020-12-11', @book, @teacher)
+  end
+
+  it 'it should return parameters' do
+    expect(@rental.date).to eq('2020-12-11')
+    expect(@rental.book.title).to eq('Common sense approach to programming')
+    expect(@rental.person.name).to eql('sam')
+  end
+
+  it 'it should return instance' do
+    expect(@rental).to be_instance_of Rental
+  end
+end
```

Comment:
- [ ] [REQUIRED] For the class `rental` you need to require the `teacher` class to fix this error that you currently have

![Screenshot from 2022-04-20 18-40-02](https://user-images.githubusercontent.com/58692590/164290763-eefe8bcd-7f41-4351-aa95-9e82365d47d8.png)
Then fix the test after that, because after I required the `teacher` class the test still failed to display this below:

![Screenshot from 2022-04-20 18-43-32](https://user-images.githubusercontent.com/58692590/164291038-5b99bcf4-460f-41cc-8e11-624e89c2f728.png)